axelrod/match.py

```diff
@@ -119,6 +119,6 @@ def play(self):
 
     def sparklines(self, c_symbol=u'█', d_symbol=u' '):
         return (
-            sparkline(self._player1.history, c_symbol, d_symbol) +
+            sparkline(self.player1.history, c_symbol, d_symbol) +
             u'\n' +
-            sparkline(self._player2.history, c_symbol, d_symbol))
+            sparkline(self.player2.history, c_symbol, d_symbol))
```

Comment:
Because it's being called in various places from outside the Player class and so shouldn't have the underscored name.